docs/conf.py

```diff
@@ -36,7 +36,7 @@
     'sphinx.ext.intersphinx',
     'sphinx.ext.napoleon',
     'sphinx.ext.extlinks',
-    'sphinxarg.ext',
+    # 'sphinxarg.ext',
```

Comment:
I'd remove this code instead of commiting commented out

Reply:
Yes, that is not needed anymore as I've documented the options manually.  That's one of the issues why I've opened it as "draft". :smile:

Reply:
I've corrected your suggestion in commit 30a5264